packages/sommelier.rb

```diff
@@ -3,7 +3,7 @@
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 programs to the built-in ChromeOS Exo Wayland server.'
   homepage 'https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/vm_tools/sommelier/'
-  version '20211129'
+  version '20220207'
```

Comment:
Should the `git_branch` match the version?

Reply:
Fixed